.rubocop.yml

```diff
@@ -5,6 +5,8 @@ AllCops:
   RunRailsCops: true
 Metrics/LineLength:
   Enabled: false
+Metrics/ClassLength:
+  Enabled: false
```

Comment:
I discovered more than one class already exceeding the limit, so it appeared to me that we were not obeying that rule.